controller/namedspaces_test.go

```diff
@@ -75,7 +76,7 @@ func (rest *TestNamedSpaceREST) TestSuccessQuerySpace() {
 		assert.Fail(t, "Failed to create an identity")
 	}
 
-	name := "Test 24"
+	name := "Test 24" + uuid.NewV4().String()
```

Comment:
@xcoulon yeah! This function should give you the name of the current Test https://golang.org/pkg/testing/#T.Name.